src/containers/Accounts/AMM/AMMAccounts/index.tsx

```diff
@@ -70,50 +63,47 @@ export const AMMAccounts = () => {
     Get the first account transaction which in this case should be AMMCreate. From this we get
     the two assets in the asset pool.
     */
-    return (
-      getAccountTransactions(rippledSocket, accountId, 1, undefined, true)
```

Comment:
The `getAccountTransactions` method can be simplified a bit now to remove the last param